nilearn/decoding/searchlight.py

```diff
@@ -186,10 +188,11 @@ def _group_iter_search_light(list_rows, estimator, X, y,
     return par_scores
 
 
-##############################################################################
-### Class for search_light ###################################################
-##############################################################################
+#
```

Comment:
Looks like you have the same version of autopep8 as Alex that removes lines of hashes like that. Try pip install -U and/or revert these changes.